### ISSUE TITLE
Eclipse installer option to clone Epsilon pulls it from git.eclipse.org

### ISSUE THREAD

**Issue by alfonsodelavega** [eclipse/epsilon#28]
When using the Eclipse installer, an option to locally clone repositories is shown at some point. If Epsilon is selected, the used urls are from "git.eclipse.org" instead of the new GitHub location.

**Comment by kolovos** [comment]
You're right, this needs fixing and the generator that produces the example project wizards under `plugins/org.eclipse.epsilon.common.dt.examples/src-gen/org/eclipse/epsilon/common/dt/examples` also needs updating to point to the GitHub repo.

**Comment by ShivangMishra** [comment]
@kolovos I want to fix this issue. Any suggestions on where I should start ?

**Comment by agarciadom** [comment]
The Eclipse installer has been fixed as of #70.
